index.js

```diff
@@ -52,6 +52,27 @@ function CleanWebpackPlugin(paths, options) {
   this.options = options;
 }
 
+/**
+ * Gives back file paths that match the given pattern
+ * 
+ * @param {string} root 
+ * @param {RegExp} pattern 
+ * @returns [string]
+ */
+function resolveRegexPaths(root, pattern) {
+  const rootChildren = fs.readdir(root);
+  
+  const resolvedPaths = [];
+
+  rootChildren.forEach(child => {
```

Comment:
The codebase is very ES5 right now. I believe we can maintain one style for now. We can later on rewrite the entire plugin in one go with ES6.

Reply:
Okay :D